File: 23_C#Basics/_06_NestedLoops/_02_MultiplicationTable/_02_MultiplicationTable.cs
using System;

public class _02_MultiplicationTable
{
    public static void Main(string[] args)
    {
        // Print multiplication table from 1 to 10
        for (int i = 1; i <= 10; i++)
        {
            for (int j = 1; j <= 10; j++)
            {
                Console.WriteLine($"{i} * {j} = {i * j}");
            }
        }
    }
}

File: 01_ProgrammingBasics/_06_NestedLoops/_02_MultiplicationTable.java
package _01_ProgrammingBasics._06_NestedLoops;

public class _02_MultiplicationTable {
    public static void main(String[] args) {
        // Print multiplication table from 1 to 10
        for (int i = 1; i <= 10; i++) {
            for (int j = 1; j <= 10; j++) {
                System.out.printf("%d * %d = %d%n", i, j, i * j);
            }
        }
    }
}
